deployment/artifacts/mainnet.json

```diff
@@ -1796,813 +1796,6 @@
         }
     },
     "137": {
-        "BqETHSubscription": {
```

Comment:
Do we need to save this entry in a separate bqeth.json file for deployment tracking purposes (with the rename of course), or is that tracked already somewhere else? Basically what you did for the marlin deployment in `new-subscription-mainnet.json`.